CHANGELOG.unreleased.md

```diff
@@ -35,6 +35,7 @@
 <!--packages-start-->
 
 - @xen-orchestra/web minor
-- xo-server patch
+- @xen-orchestra/xapi minor
+- xo-server minor
```

Comment:
IMHO, you can let `xo-server patch`

Reply:
we change the behaviour of an api call ( disabling CBT will now disabling the full chain), I think it is a more than a patch